src/valuetypes.ts

```diff
@@ -24,7 +24,7 @@ export interface IValueTypeEditor {
    * @param sampleSize
    * @return the confidence (0 ... not, 1 ... sure) that this is the right value type
    */
-  isType(name: string, index: number, data: any[], accessor: (row: any) => string, sampleSize: number): number;
+  isType(name: string, index: number, data: any[], accessor: (row: any) => string, sampleSize: number): Promise<number>;
```

Comment:
as an alternative you could delcare it as `Promise<number>|number` and use `Promise.resolve` after getting the value to ensure a number since `Promise.resolve` can handle both cases